e2e-tests/functions

```diff
@@ -19,7 +19,7 @@ namespace="${test_name}-${RANDOM}"
 conf_dir=$(realpath $test_dir/../conf || :)
 src_dir=$(realpath $test_dir/../..)
 
-if oc projects 2>&1 | egrep -q 'You have access to the following projects|You are not a member of any projects|You have one project on this server'; then
+if oc get projects ; then
```

Comment:
is it works for 3.11 ?

Reply:
@cap1984  please check openshift 3.11 and 4

Reply:
`❯ oc login https://54.247.88.117:8443 --insecure-skip-tls-verify=true
Login successful.

You have access to the following projects and can switch between them with 'oc project <projectname>':

  * default
    init-deploy-28205
    kube-public
    kube-service-catalog
    kube-system
    limits-15958
    management-infra
    openshift
    openshift-ansible-service-broker
    openshift-console
    openshift-infra
    openshift-logging
    openshift-monitoring
    openshift-node
    openshift-sdn
    openshift-template-service-broker
    openshift-web-console

Using project "default".
    ~ ────────────────────────────────────────────────────────────────────────────────────────────────── default/54-247-88-117:8443/real-admin ⎈
❯ oc get projects
NAME                                DISPLAY NAME   STATUS
default                                            Active
init-deploy-28205                                  Active
kube-public                                        Active
kube-service-catalog                               Active
kube-system                                        Active
limits-15958                                       Active
management-infra                                   Active
openshift                                          Active
openshift-ansible-service-broker                   Active
openshift-console                                  Active
openshift-infra                                    Active
openshift-logging                                  Active
openshift-monitoring                               Active
openshift-node                                     Active
openshift-sdn                                      Active
openshift-template-service-broker                  Active
openshift-web-console                              Active`

Reply:
thank youa lot